pages/overview.js

```diff
@@ -65,230 +66,220 @@ const formatByInterval = {
-        return [all, last]
-      },
-      [[]]
-    )[0]
 
-    // // WANT ALL TEASERS AS HIGH RES IMAGES?
-    //     let curls = ''
-    //     groupedTeasers.forEach(({ key, values }) => {
-    //       const path = (knownYears[year] && knownYears[year].path) || '/'
-    //       let m = `\n# ${key}\n`
-    //       m += values.map((t, i) => (
-    //         `curl -o "pngs/${swissTime.format('%Y-%m-%dT%H')(t.publishDate)}-${t.id}-${i}.png" "${getImgSrc(t, path, null, false).replace('https://cdn.repub.ch/', 'https://assets.republik.space/') + '&zoomFactor=2'}"; sleep 1;`
-    //       )).join('\n')
-    //
-    //       // console.log(m)
-    //       curls += m
-    //     })
-    //     if (typeof window !== 'undefined') { window.curls = curls }
-    //     // use copy(curls)
+        const newGroup = { key, values: [teaser] }
+        all.push(newGroup)
+        return [all, newGroup]
+      }
+      last.values.push(teaser)
+      return [all, last]
+    },
+    [[]]
+  )[0]
 
-    if (
-      !knownYears[year] ||
-      (!knownYears[year].after && !knownYears[year].path)
-    ) {
-      groupedTeasers.reverse()
-      groupedTeasers.forEach(m => m.values.reverse())
-    }
+  // // WANT ALL TEASERS AS HIGH RES IMAGES?
+  //     let curls = ''
+  //     groupedTeasers.forEach(({ key, values }) => {
+  //       const path = (knownYears[year] && knownYears[year].path) || '/'
+  //       let m = `\n# ${key}\n`
+  //       m += values.map((t, i) => (
+  //         `curl -o "pngs/${swissTime.format('%Y-%m-%dT%H')(t.publishDate)}-${t.id}-${i}.png" "${getImgSrc(t, path, null, false).replace('https://cdn.repub.ch/', 'https://assets.republik.space/') + '&zoomFactor=2'}"; sleep 1;`
+  //       )).join('\n')
+  //
+  //       // console.log(m)
+  //       curls += m
+  //     })
+  //     if (typeof window !== 'undefined') { window.curls = curls }
+  //     // use copy(curls)
 
-    return (
-      <Frame meta={meta} dark>
-        <Interaction.H1
-          style={{ color: colors.negative.text, marginBottom: 5 }}
-        >
-          {t.first(
-            [
-              `overview/${year}/title/${interval}`,
-              `overview/title/${interval}`,
-              `overview/${year}/title`,
-              'overview/title'
-            ],
-            { year }
-          )}
-        </Interaction.H1>
+  if (
+    !knownYears[year] ||
+    (!knownYears[year].after && !knownYears[year].path)
+  ) {
+    groupedTeasers.reverse()
+    groupedTeasers.forEach(m => m.values.reverse())
+  }
 
-        <P style={{ marginBottom: 10 }}>
-          {isMember
-            ? t.first([`overview/${year}/lead`, 'overview/lead'], { year }, '')
-            : t.elements(`overview/lead/${me ? 'pledge' : 'signIn'}`)}
-        </P>
-        {!isMember && (
-          <Link key='pledgeBefore' route='pledge' passHref>
-            <Button white>{t('overview/lead/pledgeButton')}</Button>
-          </Link>
+  return (
+    <Frame meta={meta} colorSchemeKey='auto'>
```

Comment:
Let's keep it dark by default for now.

```suggestion
    <Frame meta={meta} colorSchemeKey='dark'>
```